### ISSUE TITLE
Patch for /btree_map.h

### ISSUE THREAD

**Issue by GoogleCodeExporter** [linhdmn/cpp-btree#5]
```
OCD apostrophe correction ("it's" == "it is").
```

Original issue reported on code.google.com by `cartec69@gmail.com` on 1 Feb 2013 at 7:49

Attachments:
- [btree_map.h.patch](https://storage.googleapis.com/google-code-attachments/cpp-btree/issue-5/comment-0/btree_map.h.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks.  That's been fixed upstream; I'll take care of it.

```

Original comment by `josh.mac...@gmail.com` on 2 Feb 2013 at 5:30
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.0.1 and current hg.
```

Original comment by `josh.mac...@gmail.com` on 6 Feb 2013 at 6:41
- Changed state: **Fixed**
